stdlib/3/time.pyi

```diff
@@ -5,10 +5,17 @@
 # see: http://nullege.com/codes/search?cq=time
 
 import sys
-from typing import Tuple, Union
+from typing import Any, NamedTuple, Tuple, Union
 from types import SimpleNamespace
 
-TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
+if sys.version_info >= (3, 3):
+    TimeTuple = Union[
```

Comment:
Regardless of the current implementation state of the type checker, what do you consider a valid **type definition** for TimeTuple? The old one was overly restrictive.